clients/js/src/consts/networks.ts

```diff
@@ -92,8 +92,7 @@ const MAINNET = {
     rpc: "https://rpc.mainnet.near.org",
     key: getEnvVar("NEAR_KEY"),
     networkId: "mainnet",
-    deployerAccount:
-      "85957f38de1768d6db9eab29bee9dd2a01462aff9c8d83daefb9bcd2506c32d2",
+    deployerAccount: getEnvVar("NEAR_IMPLICIT_ACCOUNT_ID"),
```

Comment:
It looks like there are some typescript compiler errors with this change: https://github.com/wormhole-foundation/wormhole/actions/runs/5720831163/job/15502443893#step:4:87. Maybe we could throw if `deployerAccount` is `undefined` here: https://github.com/wormhole-foundation/wormhole/blob/f11299b888892d5b4abaa624737d0e5117d1bbb2/clients/js/src/near.ts#L162

Reply:
fixed